components/prism-bqn.js

```diff
@@ -0,0 +1,65 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:[^"]|"")*"/,
+		greedy: true,
+		alias: 'string'
+	},
+	'character-literal': {
+		pattern: /'.'/,
+		greedy: true,
+		alias: 'string'
+	},
+	'function': {
+		pattern: /•[\w¯.∞π]+[\w¯.∞π]*/i
+	},
+	'dot-notation-on-brackets': {
+		pattern: /\{(?=.*\}\.)|\}\./,
+		alias: 'namespace'
+	},
+	'special-name': {
+		pattern: /(?:𝕨|𝕩|𝕗|𝕘|𝕤|𝕣|𝕎|𝕏|𝔽|𝔾|𝕊|_𝕣_|_𝕣)/,
+		alias: 'keyword'
+	},
+	'dot-notation-on-name': {
+		pattern: /[A-Z_][\w¯∞π]*\./i,
+		alias: 'namespace'
+	},
+	'word-number-scientific': {
+		pattern: /\d+(?:\.\d+)?[e|E]¯?\d+/,
```

Comment:
The `|` in `[e|E]` is probably wrong, isn't it?

Reply:
No, both e and E work for scientific notation in BQN.

Reply:
Oh wait, that is a character set. Yes, that is wrong, I will fix it.

Reply:
Pushed Commit to that Effect.